workers.tf

```diff
@@ -153,6 +153,18 @@ resource "aws_autoscaling_group" "workers" {
     }
   }
 
+  dynamic "instance_refresh" {
+    for_each = var.worker_enable_instance_refresh ? [1] : []
```

Comment:
e.g. via:
```suggestion
    for_each = lookup(
        var.worker_groups[count.index],
        "instance_refresh_enabled",
        local.workers_group_defaults["instance_refresh_enabled"]
      ) ? [1] : []
```

Reply:
this also implies a default value set in locals.tf

Reply:
Done in my most recent commit.